system-tables/system-table-information-schema.md

```diff
@@ -192,6 +192,34 @@ CHARACTER_MAXIMUM_LENGTH: NULL
 1 row in set (0.01 sec)
 ```
 
+The description of columns in the `COLUMNS` table is as follows:
+
+* `TABLE_CATALOG`: The name of the catalog to which the table with the column belongs. The value is always `def`.
+* `TABLE_SCHEMA`: The name of the database in which the table with the column is located.
+* `TABLE_NAME`: The name of the table with the column.
+* `COLUMN_NAME`: The name of the column.
+* `ORDINAL_POSITION`: The position of the column in the table.
+* `COLUMN_DEFAULT`: The default value of the column. If the explicit default value is `NULL`, or if the column definition does not include the `default` clause, this value is `NULL`.
+* `IS_NULLABLE`: Whether the column is nullable. If the column can store empty value, this value is `YES`; otherwise, it is `NO`.
+* `DATA_TYPE`: The type of data in the column.
+* `CHARACTER_MAXIMUM_LENGTH`: The maximum length of a string column, in characters.
+* `CHARACTER_OCTET_LENGTH`: The maximum length of a string column, in bytes.
+* `NUMERIC_PRECISION`: The numeric precision of a number column.
+* `NUMERIC_SCALE`: The numeric scale of a number column.
+* `DATETIME_PRECISION`: The fractional second precision of a time column.
```

Comment:
```suggestion
* `DATETIME_PRECISION`: The fractional precision of second in the time-type column.
```

Reply:
Please check [this link](https://stackoverflow.com/questions/22594780/fractional-second-precision-in-dbmss). I'm not sure if it's a fixed term.

Reply:
```suggestion
* `DATETIME_PRECISION`: For temporal columns, the fractional seconds precision.
```